css/orbit.css

```diff
@@ -2488,42 +2488,6 @@ to use on their projects.
 	-------------------------- */
 }
 
-.NavList {
```

Comment:
Let's not make any modifications to this file since we will likely auto-generate it in the future for updates.

You can move any overrides to the 'Orbit Overrides' section at the bottom of `theme-weebly.css`.